pages/member/sign.js

```diff
@@ -0,0 +1,115 @@
+import users from "/data/users.js";
+import { isEmptyString } from "/scripts/utils.js";
+
+const errorMessage = {
+  email: {
+    empty: "이메일을 입력해주세요.",
+    valid: "올바른 이메일 주소가 아닙니다.",
+    exist: "이미 사용 중인 이메일입니다.",
+    loginFailed: "이메일을 확인해주세요.",
+  },
+  password: {
+    empty: "비밀번호를 입력해주세요.",
+    valid: "비밀번호는 영문, 숫자 조합 8자 이상 입력해 주세요.",
+    loginFailed: "비밀번호를 확인해주세요.",
+  },
+  passwordMatch: {
+    unmatched: "비밀번호가 일치하지 않아요.",
+  },
+};
+
+function handleErrorMessage(element, errorMessage = "", addOrRemoveHide) {
```

Comment:
```suggestion
function handleErrorMessage(element, addOrRemoveHide, errorMessage = "") {
```

파라미터에 기본값을 설정하면, 파라미터의 가장 뒷 부분에 위치하는 것이 일반적입니다.
이렇게 될 경우 실제 호출시 아규먼트의 생략도 가능하기 때문에
조금 더 가독성 높은 코드를 작성할 수 있습니다.